src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -37,13 +38,16 @@
     private final WPI_TalonFX mainMotor = new WPI_TalonFX(Ports.Climber.MAIN);
     private final WPI_TalonFX auxMotor = new WPI_TalonFX(Ports.Climber.AUX);
```

Comment:
We don't write motors this way as it's more easier to read that way. Furthermore, for other people who read our code, it will be a lot more easier to know how much motors we use this way.